agent/consul/client_serf.go

```diff
@@ -75,9 +85,10 @@ func (c *Client) lanEventHandler() {
 			case serf.EventUser:
 				c.localEvent(e.(serf.UserEvent))
 			case serf.EventMemberUpdate: // Ignore
+				c.nodeUpdate(e.(serf.MemberEvent))
```

Comment:
This is not a logging fix. Was this intended?

Reply:
We must of pulled something we shouldn't have. :|